Composer/packages/server/__tests__/controllers/project.test.ts

```diff
@@ -0,0 +1,149 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { Request, Response } from 'express';
+import { ProjectController } from '@src/controllers/project';
+import rimraf from 'rimraf';
+
+import { Path } from '../../src/utility/path';
+let mockRes: Response;
+
+// offer a bot project ref which to open
+jest.mock('../../src/store/store', () => {
+  const data = {
+    storageConnections: [
+      {
+        id: 'default',
+        name: 'This PC',
+        type: 'LocalDisk',
+        path: '.',
+        defaultPath: '.',
+      },
+    ],
+    recentBotProjects: [] as any[],
+  } as any;
+  return {
+    Store: {
+      get: (key: string) => {
+        return data[key];
+      },
+      set: (key: string, value: any) => {
+        data[key] = value;
+      },
+    },
+  };
+});
```

Comment:
This should be extracted to a mock file instead. https://jestjs.io/docs/en/manual-mocks#mocking-user-modules